misc/uuid.c

```diff
@@ -61,8 +61,8 @@
  */
 
 #include "uuid.h"
-#include "error.h"
-#include "avstring.h"
+#include "libavutil/error.h"
+#include "libavutil/avstring.h"
```

Comment:
I don't think it makes sense to keep the meson and this commit apart
not like one works without the other

Reply:
My preference was to add the copied files completely unchanged. In principle that makes it easier to track what we changed if you'd ever wanted to merge in further updates. It's not going to happen in this case, but as the added files just sit there without the meson changes, it also doesn't stop compilation at that commit - so no harm.